src/redux/items/thunks.js

```diff
@@ -59,3 +59,11 @@ export const rateItemAsync = createAsyncThunk(
     return response.data.result
   }
 )
+
+export const applySearchNameAsync = createAsyncThunk(
+  'items/search',
+  async (searchText) => {
+	  const response = await axios.get(`${URL}/search/${searchText}`)
+	  return response.data.result
+  }
+)
```

Comment:
Will change the naming, but I feel search by description may not be that necessary since it may not contain important information that worth for the users to search for. 
And the filters and the search bar will be applied separately so I don't think there will be other fields implemented afterwards. 
But open for other possibilities, so let me know.